internal/tui/termsafe_test.go

```
@@ -0,0 +1,160 @@
+// renderer degrades to plain text. That is a property of the harness, not of
+// this change — so it is asserted rather than assumed. Without it, a screen
+// full of styling escapes would fail the hostile-fixture tests below for
+// reasons having nothing to do with a session name, and would keep failing
+// after a correct fix.
+func TestViewOfBenignFixtureIsAlreadyInert(t *testing.T) {
+	fake := &exec.FakeRunner{RunFunc: func(_ string, _ []string) (string, error) {
+		return oneSessionRow, nil
+	}}
+	m := sized(newModel(context.Background(), tmux.New(fake), true), 80, 24)
+	out, _ := m.Update(key("2"))
+	m = out.(model)
```

golangci-lint (2.12.2)

[error] type assertion must be checked

(forcetypeassert)